cves/kernel/CVE-2020-10781.yml

```diff
@@ -278,9 +274,9 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
-discussion:
+  answer:  true
+  note:  This driver could create ZRAM device nodes in the /dev/ directory, which is the basis for this vulnerability.
```

Comment:
Explain what are ZRAM device nodes. Also, explain how it uses inter-process communication

Reply:
Added a note about what ZRAM is and why it's important :)